src/plugins/stacked-barchart/src/StackedBarchartPanel.tsx

```diff
@@ -0,0 +1,264 @@
+      .append('g')
+      .attr('transform', `translate(${margin.left}, ${svgHeight - margin.bottom})`)
+      .call(xAxisGenerator)
+      .selectAll('text')
+      .style('fill', theme.colors.text);
+
+    svg
+      .append('text')
+      .attr('transform', `translate(${(width - margin.right + margin.left) / 2}, ${svgHeight - 8})`)
+      .style('fill', theme.colors.text)
+      .style('text-anchor', 'middle')
+      .text(options.xLabel);
+
+    // Y axis
+    const yAxis = d3
+      .scaleLinear()
+      .domain([0, xMaxGroupCount])
+      .range([svgHeight - margin.bottom - margin.top, 0]);
+
+    svg
+      .append('g')
+      .attr('transform', `translate(${margin.left}, ${margin.top})`)
+      .call(d3.axisLeft(yAxis))
+      .selectAll('text')
+      .style('fill', theme.colors.text);
+
+    svg
+      .append('text')
+      .attr('transform', 'rotate(-90)')
+      .attr('x', -(svgHeight - margin.bottom + margin.top) / 2)
+      .attr('y', 15)
+      .style('text-anchor', 'middle')
+      .style('fill', theme.colors.text)
+      .text(options.yLabel);
+
+    // Event listeners
+    const mouseover = function (this: d3.BaseType | SVGElement, event: any) {
+      const subgroupName = (d3.select((this as SVGElement).parentNode as Element).datum() as { key: string }).key;
+      ReactDOM.render(
+        <>
+          {yDistinctGroups.map((item) => {
+            return (
+              <div
+                key={item}
+                className={css`
+                  display: flex;
+                  justify-content: space-between;
+                  color: ${subgroupName === item ? theme.colors.textStrong : theme.colors.text};
+                `}
+              >
+                <div>
+                  <div
+                    className={css`
+                      background-color: ${colorScale(item) as string};
+                      border-radius: 1px;
+                      display: inline-block;
+                      height: 4px;
+                      margin-right: 8px;
+                      width: 14px;
+                    `}
+                  />
+                  <div
+                    className={css`
+                      font-size: 12px;
+                      display: inline-block;
+                      font-weight: ${subgroupName === item ? 500 : 200};
+                    `}
+                  >
+                    {item}:
+                  </div>
+                </div>
+                <div
+                  className={css`
+                    float: right;
+                    padding-left: 15px;
+                    font-weight: 500;
+                  `}
+                >
+                  {event.data[item]}
+                </div>
+              </div>
+            );
+          })}
+        </>,
+        graphTooltipRef.current
+      );
+      tooltip.style('display', 'block');
+      svg.selectAll('.myRect').style('opacity', 0.2);
+      svg.selectAll('.' + subgroupName).style('opacity', 1);
```

Comment:
Can you add the video timecode to the tooltip?

Reply:
Good Idea) Added the group name to the tooltip)